flake.nix

```
@@ -46,12 +47,23 @@
       };
       # --------------------------------------------------------------
 
+      # --- What CI should build -------------------------------------
+      x86_64-linux.required = inputs.nixpkgs.legacyPackages.x86_64-linux.releaseTools.aggregate {
+        name = "required for CI";
+        constituents = __attrValues x86_64-linux.x86_64-linux;
```

```suggestion
        constituents = __attrValues x86_64-linux.x86_64-linux ++ __attrValues x86_64-linux.x86_64-windows;
```
:pray: We need those, too (they are cross-built from Linux to Windows)